src/commands/createFunction/FunctionSubWizard.ts

```diff
@@ -86,7 +90,25 @@ export class FunctionSubWizard {
                     break;
             }
 
-            if ((!template.isHttpTrigger && !template.isSqlBindingTemplate) && !canValidateAzureWebJobStorageOnDebug(context.language) && !await getAzureWebJobsStorage(context, context.projectPath)) {
+            switch (context.newDurableStorageType) {
+                case DurableBackend.Netherite:
+                    promptSteps.push(new EventHubsConnectionPromptStep(), new NetheriteEventHubNameStep());
+                    executeSteps.push(new EventHubsConnectionExecuteStep(), new NetheriteConfigureHostStep());
+                    break;
+                case DurableBackend.SQL:
+                    // Todo: Uncomment out in future PR
+                    // promptSteps.push(new SqlDatabaseConnectionPromptStep(), new SqlDatabaseListStep());
+                    // executeSteps.push(new SqlDatabaseConnectionExecuteStep());
+                    break;
+                case DurableBackend.Storage:
+                default:
+            }
+
+            if (
+                !await getLocalConnectionString(context, ConnectionKey.Storage, context.projectPath) &&   // If we already have a storage setup, don't re-prompt
+                (context.newDurableStorageType ||   // Always prompt for new durable storage setups unless a storage account has already been linked
+                    (!template.isHttpTrigger && !template.isSqlBindingTemplate && !canValidateAzureWebJobStorageOnDebug(context.language)))
+            ) {
```

Comment:
Would appreciate a second pair of 👀 here to make sure I preserved the logic properly!  I did my best to make this statement more readable and and tried to organize it in a way that makes sense.

Reply:
Yeah, I believe we should be doing it for all new DF storage types